build_tools/build_libtorch.sh

```diff
@@ -127,11 +149,38 @@ build_pytorch() {
     # STATIC_DISPATCH_BACKEND=ON
     # BUILD_LITE_INTERPRETER=OFF
   fi
-  BUILD_SHARED_LIBS=${BUILD_SHARED_LIBS} USE_LIGHTWEIGHT_DISPATCH=${USE_LIGHTWEIGHT_DISPATCH} STATIC_DISPATCH_BACKEND=${STATIC_DISPATCH_BACKEND} BUILD_LITE_INTERPRETER=${BUILD_LITE_INTERPRETER} BUILD_TEST=OFF USE_GLOO=OFF USE_MPS=OFF USE_PYTORCH_QNNPACK=OFF USE_OPENMP=OFF  USE_OBSERVERS=OFF USE_KINETO=OFF USE_EIGEN_FOR_BLAS=OFF CMAKE_CXX_FLAGS="-D_GLIBCXX_USE_CXX11_ABI=${CXX_ABI}" USE_FBGEMM=OFF USE_NCCL=OFF INTERN_DISABLE_ONNX=OFF USE_CUDA=OFF USE_MKL=OFF USE_XNNPACK=OFF USE_DISTRIBUTED=OFF USE_BREAKPAD=OFF USE_MKLDNN=OFF USE_QNNPACK=OFF USE_NNPACK=OFF ONNX_ML=OFF CMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES} python setup.py  bdist_wheel -d "$WHEELHOUSE"
+  BUILD_LITE_INTERPRETER=${BUILD_LITE_INTERPRETER} \
+  BUILD_SHARED_LIBS=${BUILD_SHARED_LIBS} \
+  BUILD_TEST=OFF \
+  CMAKE_CXX_FLAGS="-D_GLIBCXX_USE_CXX11_ABI=${CXX_ABI}" \
+  CMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES} \
+  MAX_JOBS=${MAX_JOBS} \
+  ONNX_ML=OFF \
+  STATIC_DISPATCH_BACKEND=${STATIC_DISPATCH_BACKEND} \
+  USE_BREAKPAD=OFF \
+  USE_CUDA=OFF \
+  USE_DISTRIBUTED=OFF \
+  USE_EIGEN_FOR_BLAS=OFF \
+  USE_FBGEMM=OFF \
+  USE_GLOO=OFF \
+  USE_KINETO=OFF \
+  USE_LIGHTWEIGHT_DISPATCH=${USE_LIGHTWEIGHT_DISPATCH} \
+  USE_MKL=OFF \
+  USE_MKLDNN=OFF \
+  USE_MPS=OFF \
+  USE_NCCL=OFF \
+  USE_NINJA=${USE_NINJA} \
+  USE_NNPACK=OFF \
+  USE_OBSERVERS=OFF \
+  USE_OPENMP=OFF \
+  USE_PYTORCH_QNNPACK=OFF \
+  USE_QNNPACK=OFF \
+  USE_XNNPACK=OFF \
+  python setup.py bdist_wheel -d "$WHEELHOUSE"
 }
 
 package_pytorch() {
-  mkdir -p libtorch/{lib,bin,include,share}
+  rm -rf libtorch/{lib,bin,include,share}
```

Comment:
I'm not sure about this - on my M1 the `mv` commands below failed because these directories already existed.

Reply:
I moved this from `cp -rv` to `mv` here so the GH runners don't take a long time. Just FYI on why it is different from upstream.